static/doc.json

```diff
@@ -2569,6 +2682,26 @@
             "description": "The list of materials required to craft the item."
           }
         }
+      },
+      "NHEvent": {
+        "type": "object",
+        "properties": {
+          "date": {
+            "type": "date",
+            "example": "2021-05-01",
+            "description": "The date of the event."
+          },
+          "event": {
+            "type": "string",
+            "example": "May Day event begins",
+            "description": "The description of the event."
+          },
+          "event_name": {
```

Comment:
We should call this `event_url` and have it be an actual link to the article. `event_name` isn't really accurate since, since the link column has things like "Nook Shopping seasonal event".